core-feature-pack/pom.xml

```diff
@@ -112,6 +112,11 @@
 
         <dependency>
             <groupId>org.jboss.logging</groupId>
+            <artifactId>jboss-logging-annotations</artifactId>
```

Comment:
I meant more should it really be in the feature pack at all? It is a build time dependency only afaik used to generate the logger class impls. So subsystems and other modules that need logging use my example